packages/eslint-config-hudl/docs/comparisons/with-airbnb-master.md

```diff
@@ -34,209 +34,1232 @@ should be added to the list in [the diff task](../../tasks/docs/diffs/diffs-to-r
+-     react/no-array-index-key
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-children-prop
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-comment-textnodes
+  - [<ins>&quot;off&quot;</ins>]
+-     react/no-danger-with-children
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-find-dom-node
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-render-return-value
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-unescaped-entities
+  - [<ins>&quot;error&quot;</ins>]
+-     react/no-unused-prop-types
+  - [<ins>&quot;error&quot;</ins>, <ins>{&quot;customValidators&quot;:[],&quot;skipShapeProps&quot;:true}</ins>]
+-     react/prefer-stateless-function
+  - [<ins>&quot;error&quot;</ins>]
+-     react/require-default-props
+  - [<ins>&quot;error&quot;</ins>]
+-     react/require-optimization
+  - [<ins>&quot;off&quot;</ins>, <ins>{&quot;allowDecorators&quot;:[]}</ins>]
+-     react/require-render-return
+  - [<ins>&quot;error&quot;</ins>]
+-     react/sort-prop-types
+  - [<ins>&quot;off&quot;</ins>, <ins>{&quot;ignoreCase&quot;:true,&quot;callbacksLast&quot;:false,&quot;requiredFirst&quot;:false}</ins>]
+-     react/style-prop-object
+  - [<ins>&quot;error&quot;</ins>]
+-     react/wrap-multilines
+  - [<ins>&quot;off&quot;</ins>]
+- 
+    [require-await](http://eslint.org/docs/rules/require-await.html)
+  - [<ins>&quot;off&quot;</ins>]
+- 
+    [require-jsdoc](http://eslint.org/docs/rules/require-jsdoc.html)
+  - [<ins>&quot;off&quot;</ins>]
+- 
+    [rest-spread-spacing](http://eslint.org/docs/rules/rest-spread-spacing.html)
+  - [<ins>&quot;error&quot;</ins>, <ins>&quot;never&quot;</ins>]
+- 
+    [sort-imports](http://eslint.org/docs/rules/sort-imports.html)
+  - [<ins>&quot;off&quot;</ins>, <ins>{&quot;ignoreCase&quot;:false,&quot;ignoreMemberSort&quot;:false,&quot;memberSyntaxSortOrder&quot;:[&quot;none&quot;,&quot;all&quot;,&quot;multiple&quot;,&quot;single&quot;]}</ins>]
+- 
+    [sort-keys](http://eslint.org/docs/rules/sort-keys.html)
+  - [<ins>&quot;off&quot;</ins>, <ins>&quot;asc&quot;</ins>, <ins>{&quot;caseSensitive&quot;:false,&quot;natural&quot;:true}</ins>]
+- 
+    [symbol-description](http://eslint.org/docs/rules/symbol-description.html)
+  - [<ins>&quot;error&quot;</ins>]
+- 
+    [template-curly-spacing](http://eslint.org/docs/rules/template-curly-spacing.html)
+  - [<ins>&quot;error&quot;</ins>]
+- 
+    [template-tag-spacing](http://eslint.org/docs/rules/template-tag-spacing.html)
+  - [<ins>&quot;off&quot;</ins>, <ins>&quot;never&quot;</ins>]
+- 
+    [unicode-bom](http://eslint.org/docs/rules/unicode-bom.html)
+  - [<ins>&quot;error&quot;</ins>, <ins>&quot;never&quot;</ins>]
+- 
+    [yield-star-spacing](http://eslint.org/docs/rules/yield-star-spacing.html)
+  - [<ins>&quot;error&quot;</ins>, <ins>&quot;after&quot;</ins>]
 
 #### Removed rules
 _None_
 
 #### Edited rules
 - 
+    [accessor-pairs](http://eslint.org/docs/rules/accessor-pairs.html)
+  - [<del>0</del>]
+  - [<ins>&quot;off&quot;</ins>]
```

Comment:
It would be nice to update our rules to use the preferred word form (`'off'` vs `0`) to reduce the non-semantic differences between our configs.